### PR TITLE
Route create booksuggestion

### PR DESCRIPTION
**Pull request by Sspadea**
## Summary

- Route create book suggestion

## Trello Card

https://trello.com/c/CLEfey0g/25-1-crear-ruta-para-creaci%C3%B3n-de-un-booksuggestion

## Screenshots or Screencasts

Captura de Pantalla 2021-06-22 a la(s) 17 06 29

### CODE REVIEW

**Review by blacksam07 — APPROVED** [review]

File: app/controllers/api/v1/book_suggestions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApplicationController
+      def create
+        book_suggestion = BookSuggestion.new(book_suggestion_params)
+        book_suggestion.user = current_user if user_signed_in?
+        if book_suggestion.save
+          render json: book_suggestion, status: :created
+        else
+          render json: { errors: [{ code: '422', message: book_suggestion.errors }] }, status: :unprocessable_entity
```
Comment (redwarewolf) [inline app/controllers/api/v1/book_suggestions_controller.rb:10]: I wouldn't say this is an unprocessable_entity.
Also move logic to a service.

File: app/helpers/book_suggestions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module BookSuggestionsHelper
+end
```
Comment (redwarewolf) [inline app/helpers/book_suggestions_helper.rb:2]: Why is this empty?

File: spec/helpers/book_suggestions_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# describe BookSuggestionsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe BookSuggestionsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (redwarewolf) [inline spec/helpers/book_suggestions_helper_spec.rb:15]: whats with this file?

File: spec/controllers/book_suggestions_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe BookSuggestionsController, type: :controller do
```
Comment (redwarewolf) [inline spec/controllers/book_suggestions_controller_spec.rb:3]: shoulda matchers

**Review by redwarewolf — CHANGES_REQUESTED** [review]

**Pull request merged**